src/js/tracker.js

```diff
@@ -1410,7 +1413,8 @@
 
 				// Periodic check for activity.
 				lastActivityTime = now.getTime();
-				setInterval(function heartBeat() {
+				clearInterval(pagePingInterval)
+				pagePingInterval = setInterval(function heartBeat() {
```

Comment:
LGTM (just add a semicolon after `clearInterval`). Do you have evidences it was invoked multiple times?